framework/src/geomsearch/NearestNodeLocator.C

```diff
@@ -93,8 +93,7 @@ NearestNodeLocator::findNodes()
     // This means there was a user specified inflation... so we can build a BB
     if (inflation.size() > 0)
     {
-      BoundingBox my_box =
-          MeshTools::create_local_bounding_box(_mesh);
+      BoundingBox my_box = MeshTools::create_local_bounding_box(_mesh);
```

Comment:
clang-format has a "target" line length, and it will both lengthen (by removing newlines) and shorten lines in order to try and match this target. Your changes to this line must have brought it under the target length. 

It's unfortunate; I would prefer a max instead of a target but this does not seem to be possible currently.